BurnerB/PropertyProLite-CH2
# Ft post advert 166858131

#### What does this PR do?
Create post property endpoint
#### Description of Task to be completed?
* Write post property models,controllers and set the route
* Tests should pass
#### How should this be manually tested?
CLONE this repo
CD into it
run `npm test`
Tests should pass

using postman run the link:
http://127.0.0.1:5000/api/v1/property
POST with the data:
```
{
,“status” ​:​ String​
,“price” :Float​
,“state” :String​
,“city” ​:​ ​String​
,“address” ​:​ ​String​
,“type” ​:​ ​String​
,“image_url” ​:​ ​String​
}
```

#### What are the relevant pivotal tracker stories?
#166858131
#### Screenshots (if appropriate)
![post-pproperty success](https://user-images.githubusercontent.com/45785786/60335406-86ff2600-99a6-11e9-966e-115e6055d72b.png)
![postproperty passing](https://user-images.githubusercontent.com/45785786/60335427-8c5c7080-99a6-11e9-9be5-60bdfcd4597f.png)
